api/swagger.yaml

```diff
@@ -4443,6 +4443,8 @@ paths:
       responses:
         200:
           description: "no error"
+        404:
+          description: "repository not found"
```

Comment:
update looks better although `does not exist or no read access` is already poor grammar, but maybe update the description to match `repository does not exist or no read access`